Arm64v8

dalir opened the pull request.

Commit:
Adding alpine-aarch64 support for jnlp-slave

Commit:
leave the docker installation to apk rather than manual installation.

blaines commented on alpine-arm64v8/Dockerfile:
```diff
@@ -0,0 +1,108 @@
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.7/community' >> /etc/apk/repositories \
+  && apk --update add make \
+  bash \
+  docker \
```
Can we confirm the version here? 17.06.0-ce or better

dalir replied:
17.10.0

dalir replied:
Thought it's not a bad idea to have always the latest stable in the build

blaines commented on alpine-arm64v8/Dockerfile.build:
```diff
@@ -0,0 +1,50 @@
+FROM alpine:3.6
```
rm this file

dalir replied:
will do.

blaines reviewed (changes requested):
We should remove the .build files, also add a Makefile for automated build of the images.

Commit:
removing the docker.build files as they were not in use.

Commit:
Adding make file to the repository.
TODO: Should make the change to makefile in a way that it builds for specific variant and Jenkinsfile runs variants in parallel instead

Commit:
Passing version of the Docker as part of the argument to the makefile into Dockerfile.

blaines commented on alpine-arm64v8/Dockerfile:
```diff
@@ -0,0 +1,109 @@
+
+# Root user is required to interact with Docker via docker.sock
+USER root
+ARG DOCKER_VERSION 17.10.0-r0
```
I believe you forgot an =

blaines commented on alpine/Dockerfile:
```diff
@@ -23,56 +23,15 @@ FROM jenkinsci/jnlp-slave:alpine
-ENV DOCKER_COMPOSE_VERSION 1.16.1
+
 ENV SAM_VERSION 0.2.4
+ARG DOCKER_VERSION 17.10.0-r0
```
Same as other comment "="

Commit:
Missing assignment operator for ARG.

Commit:
Adding the repository to jenkins
